justfile

```
@@ -114,13 +127,15 @@ build-windows-arm64:
     -ldflags="-s -w" \
     -ldflags="-X 'github.com/open-sauced/pizza-cli/pkg/utils.Version=${VERSION}'" \
     -ldflags="-X 'github.com/open-sauced/pizza-cli/pkg/utils.Sha=$(git rev-parse HEAD)'" \
+    -ldflags="-X 'github.com/open-sauced/pizza-cli/pkg/utils.Datetime=${DATETIME}'" \
     -o build/pizza-${GOOS}-${GOARCH}
 
 # Builds the container and marks it tagged as "dev" locally
 build-container:
   docker build \
     --build-arg VERSION=$(git describe --tags --always) \
     --build-arg SHA=$(git rev-parse HEAD) \
+    --build-arg DATETIME=$(date -u +"%Y-%m-%d %H:%M:%S") \
```

The date command has problems with terminating the string within the shell script. So, I'd recommend wrapping those in `""`:

```suggestion
    --build-arg VERSION="$(git describe --tags --always)" \
    --build-arg SHA="$(git rev-parse HEAD)" \
    --build-arg DATETIME="$(date -u +'%Y-%m-%d %H:%M:%S')" \
```